db/migrations/collector/20240109170034_fcd_tx_log.down.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+DROP TABLE IF EXISTS "fcd_tx_log";
+
+DROP INDEX IF EXISTS fcd_tx_log_timestamp_idx;
```

Comment:
`DROP TABLE` is enough in this case

Reply:
but rather `DROP SEQUENCE` is needed. why don't you use bigserial instead of sequence?